src/ci/bin/testCentaurEngineUpgradeLocal.sh

```diff
@@ -6,11 +6,15 @@ export CROMWELL_BUILD_OPTIONAL_SECURE=true
 # shellcheck source=/dev/null
 source "${BASH_SOURCE%/*}/test.inc.sh" || source test.inc.sh
 
-cromwell::build::setup_common_environment
+if [ "${CROMWELL_BUILD_PROVIDER}" = "${CROMWELL_BUILD_PROVIDER_TRAVIS}" -a  -n "${TRAVIS_PULL_REQUEST_BRANCH}" ]; then
```

Comment:
Should we sanity-check that these magic environment variables also exist here too?

Reply:
These other variables are much more fundamental to the operation of the script, if they're not set I think that fact would be impossible to ignore. 🙂